IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterways/Entities/IfcConveyorSegment/DocEntity.xml

```diff
@@ -2,7 +2,8 @@
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcConveyorSegment" UniqueId="5861065a-e519-4b96-8adc-7d4cc6c7e95e" BaseDefinition="IfcFlowSegment" EntityFlags="32">
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="cf423959-385f-4bf2-bbe3-355dee7ff725" DefinedType="IfcConveyorSegmentTypeEnum" AttributeFlags="1">
-			<Documentation></Documentation>
+			<Documentation>Identifies the predefined type of a conveyor segment from which the type modelled, may be set. This type may associate additional specific property sets.
```

Comment:
Could be simplified as it is in existing PDT descriptions. See #43

Reply:
sorted